src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -25,11 +25,11 @@
     {% if prepend or append %}
         <div class="input-group">
             {% if prepend %}
-                <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
+                <span class="input-group-addon">{{ money_pattern|form_encode_currency }}</span>
```

Comment:
`bootstrap_base_layout.html.twig + bootstrap_4_layout.html.twig` as of 3.4 instead